### PR TITLE
timezone-data: convert to new tzdata and tzcode packages

### PR DESCRIPTION
**Pull request by lrusak**
This is the best I could do to make it so we don't have to pack the sources ourselves, now we can just bump the version when a new one is released.

Not sure if I should update the licence header.

Is probably worth a test build or two ;)

### CODE REVIEW

**Comment by CGarces** [comment]
libc has a dependency with tzdata
But no one has a dependency over tzcode?

**Comment by stefansaraev** [comment]
hint hint https://github.com/stefansaraev/TB/blob/master/packages/base/tz/package.mk

**Comment by lrusak** [comment]
or that

**Comment by stefansaraev** [comment]
https://github.com/eggert/tz is an official source btw. tzcode and tzdata tarbals are based in this source.

**Comment by lrusak** [comment]
updated, thanks @stefansaraev

File: packages/sysutils/tz/package.mk [hunk 1]
```diff
@@ -16,41 +16,34 @@
...
   mkdir -p $INSTALL/usr/share/zoneinfo
-    mv $INSTALL/etc/zoneinfo/* $INSTALL/usr/share/zoneinfo
+  mv $INSTALL/etc/zoneinfo/* $INSTALL/usr/share/zoneinfo
 
-  rm -rf $INSTALL/man
   rm -rf $INSTALL/etc
-
   mkdir -p $INSTALL/etc
-    ln -sf /var/run/localtime $INSTALL/etc/localtime
+  ln -sf /storage/.cache/localtime $INSTALL/etc/localtime
```
Comment (stefansaraev) [inline packages/sysutils/tz/package.mk]: keep this as it was.

**Comment by MilhouseVH** [comment]
Good to go? I've not seen any issues reported.

**Comment by lrusak** [comment]
Yep

**Pull request merged**